server/src/test/java/io/crate/metadata/cluster/AlterTableClusterStateExecutorTest.java

```diff
@@ -116,54 +106,4 @@ public void test_group_settings_are_not_filtered_out() throws IOException {
         assertThat(filteredSettings.isEmpty()).isFalse();
         assertThat(filteredSettings.get(fullName)).isEqualTo("node1");
     }
-
-    @Test
```

Comment:
this is all related to family of bugs "smth is forgotten to copy or copied partly before applying delta". Now we don't use this for alter column so tests are outdated